pages/index.html

```diff
@@ -4,9 +4,38 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <title>Katina Carranza Personal Portfolio</title>
 </head>
 <body>
-  
+  <h1>
+    My Personal Portfolio
+  </h1>
+  <section class="container">
+  <div>
+    <h2>
+        Home
+    </h2>
+    <h2>
+      <a href="about.html">About</a>
+    </h2>
+    <h2>
+      <a href = "portfolio.html">Portfolio</a>
+    </h2>
+</div>
+    <img 
+      alt= "Baby Photos" src = "/images/Baby-Photos.jpg" >
+    <div class = "purple" >
```

Comment:
instead of div tags you want to use section tags here. Here is a link to an article describing the difference between the two. https://www.geeksforgeeks.org/what-is-the-difference-between-section-and-div-tags-in-html/